### PR TITLE
Fix - Concurrent jobs launch: start several cron expressions at the same time

### PR DESCRIPTION
**Pull request by yevhen-hryhorevskyi**
Hi there,

I liked the plugin and idea of setting up multiple cron expressions with different parameters. I looked through readme and found an image where two jobs (#18, #19) were launched at the same time. That was exactly what I needed.

But, unfortunately, I was not able to setup configuration to run different periodical cron expressions at the same time.
Digging into the code, I realized that this plugin worked differently:
- It runs a check every minute to launch a cron task with specific parameters.
- It makes a lookup only for the first cron expression that should start at that moment.

Based on the documentation and real implementation, I concluded that there was a bug in code, or it was the desired behavior and misleading documentation.

### CODE REVIEW

**Comment by danielfariati** [comment]
I'm having the exact same issue described / solved in this pull-request.
It would be awesome if someone can merge this.

**Comment by carlosaya** [comment]
same problem here!  can this pull request get some love?   🙏

**Comment by wy-z** [comment]
I'm having the same issue too, hope this PR can be merged.

**Comment by BenCamareno** [comment]
Same issue as described, please merge

**Comment by batmat** [comment]
Hi everyone,

I'm the current maintainer of this plugin. TBH with myself, I lack the time to maintain this plugin, so if somebody can take over the maintenance, I'll be happy to hand it over.

Process: https://www.jenkins.io/doc/developer/plugin-governance/adopt-a-plugin/

Thanks!

File: src/test/java/org/jenkinsci/plugins/parameterizedscheduler/ParameterizedCronTabListTest.java
Comment (res0nance) [inline src/test/java/org/jenkinsci/plugins/parameterizedscheduler/ParameterizedCronTabListTest.java:31]: This second test should be split into a new test

**Review by res0nance — CHANGES_REQUESTED** [review]

**Comment by res0nance** [comment]
@yevhen-hryhorevskyi are you still interested in this PR?

**Comment by yevhen-hryhorevskyi** [comment]
@res0nance Considering the number of comments to merge this PR, yes I'm still interested in it. Let me take care of the comments & merge conflicts later today

**Comment by res0nance** [comment]
> @res0nance Considering the number of comments to merge this PR, yes I'm still interested in it. Let me take care of the comments & merge conflicts later today

Do you have time to still work on this? Would you like me to continue with this?

**Comment by chunming08** [comment]
> > @res0nance Considering the number of comments to merge this PR, yes I'm still interested in it. Let me take care of the comments & merge conflicts later today
> 
> Do you have time to still work on this? Would you like me to continue with this?

@res0nance , can you please kindly share the schedule of the next release of this plugin? It's been 4 months since last and I am still waiting for the functionality that this PR provides - https://github.com/jenkinsci/parameterized-scheduler-plugin/pull/47

**Comment by res0nance** [comment]
> > > @res0nance Considering the number of comments to merge this PR, yes I'm still interested in it. Let me take care of the comments & merge conflicts later today
> > 
> > 
> > Do you have time to still work on this? Would you like me to continue with this?
> 
> @res0nance , can you please kindly share the schedule of the next release of this plugin? It's been 4 months since last and I am still waiting for the functionality that this PR provides - #47

Probably releasing this today because i just wrote PR #105 to supersede this. Once merged there shouldn't be anything else outstanding.

**Pull request closed without merging**
